mobile/src/libs/packages/store/store.package.ts

```diff
@@ -35,14 +42,16 @@ class Store {
       devTools: config.ENV.APP.ENVIRONMENT !== AppEnvironment.PRODUCTION,
       reducer: {
         auth: authReducer,
+        notification: notificationReducer,
       },
-      middleware: (getDefaultMiddleware) => {
-        return getDefaultMiddleware({
+      middleware: (getDefaultMiddleware) => [
+        ...getDefaultMiddleware({
```

Comment:
Let's use .concat here
getDefaultMiddleware({...}).concat([handleError])

Reply:
@fshabanov if I use .concat() I got a linter error
```typescript
   public constructor(config: Config) {
    this.instance = configureStore({
      devTools: config.ENV.APP.ENVIRONMENT !== AppEnvironment.PRODUCTION,
      reducer: {
        auth: authReducer,
        notification: notificationReducer,
      },
      middleware: (getDefaultMiddleware) =>
        getDefaultMiddleware({
          thunk: {
            extraArgument: this.extraArguments,
          },
        }).concat(handleError),
    });
  }
```
How should I do it properly??

Reply:
@BrianSammit I think you missed square brackets around 'handleError'. It should be 
```.concat([handleError])```

Reply:
@fshabanov Still getting the linter error 
```typescript
  public constructor(config: Config) {
    this.instance = configureStore({
      devTools: config.ENV.APP.ENVIRONMENT !== AppEnvironment.PRODUCTION,
      reducer: {
        auth: authReducer,
        notification: notificationReducer,
      },
      middleware: (getDefaultMiddleware) =>
        getDefaultMiddleware({
          thunk: {
            extraArgument: this.extraArguments,
          },
        }).concat([handleError]),
    });
  }
```
![Screenshot 2023-08-25 135533](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/64173189/58e6cfe6-aa6e-48cb-a69a-17fffaf6709e)

Reply:
@BrianSammit in that case let's use spread operator for now